server/controllers/lecture-controller.js

```diff
@@ -0,0 +1,38 @@
+const moment = require("moment");
+const knex = require("./../db");
+
+
+//Get all available bookings by one student before deadline
+exports.getBookingLectures = async (req, res) => {
+    const studentId = req.user && req.user.id;  //studente autenticato
```

Comment:
User should not be passed as parameter, we know who the user is since the user is authenticated.

Reply:
Remove Italian comment.